app/assets/stylesheets/common.scss.erb

```diff
@@ -13,7 +13,7 @@
   font-family: 'Source Sans Pro', 'Arial Unicode MS', Helvetica, sans-serif;
   font-size: $font-size-base;
   overflow-y: hidden;
-  @include shadow(3);
+  border: 1px solid #5f574F;
```

Comment:
We shouldn't reference colors directly in our css declarations.  This is a color that is defined in SUL Styles as `$color-pantone-405` (which we have defined as `$sul-border-color` in SUL Embed).  Can we use that variable here instead if the color itself?